pax-url-aether/src/main/java/org/ops4j/pax/url/mvn/internal/config/MavenConfigurationImpl.java

```diff
@@ -582,6 +583,11 @@ private void parseProxiesFromProperty(String proxySettings, Map<String, Map<Stri
 
     private String getSettingsPath() {
         URL url = getSettingsFileUrl();
+        try {
+            return url == null ? null : new File(url.toURI().getSchemeSpecificPart()).getAbsolutePath();
+        } catch (URISyntaxException e) {
+        }
+		
         return url == null ? null : url.getPath();
```

Comment:
Could you place the return in the catch clause, because you cannot leave a catch close empty like that.
It'll be more readable.
Also this does not need to check for null as it is handled by the *try* code.